src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1185,30 +1207,44 @@
 helps['network nic'] = """
     type: group
     short-summary: Manage network interfaces.
+    long-summary: >
+        To learn more about network interfaces in Azure visit https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-network-interface
 """
 
 helps['network nic show-effective-route-table'] = """
     type: command
     short-summary: Show all route tables applied to a network interface.
+    long-summary: >
+        To learn more about how to troubleshoot using effective route tables visit https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-routes-troubleshoot-portal#using-effective-routes-to-troubleshoot-vm-traffic-flow
```

Comment:
The wording of the `long-summary` and the wording of `short-summary` regarding the command display doesn't match. There doesn't appear to be any clear differentiation between "effective route" and "route" in documentation, so consistent wording should be used in the inline help, and match the appropriate term for the product.

Reply:
Fixed on my branch